src/main/java/core/basesyntax/strategy/impl/BalanceOperationHandler.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.enums.Operation;
+import core.basesyntax.strategy.mapvalidator.KeyAbsentValidator;
+import java.util.Map;
+
+public class BalanceOperationHandler
+        extends AbstractOperationHandler
+        implements KeyAbsentValidator {
+    public BalanceOperationHandler() {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the constructor.